src/Documentation/doc/advanced/corporations.md

```diff
@@ -1,3 +1,77 @@
 # Corporations
 
-PLACEHOLDER
+- [Basic gameplay and term](./corporation/basic-gameplay-and-term.md)
```

Comment:
Optional: An introductory paragraph would probably be good here, before launching into the table of contents. However, it's not wholly needed, nor am I asking you to write one (unless you feel inspired to) - this is half just a note to myself.

Reply:
I added it in [#2f285a5 ](https://github.com/bitburner-official/bitburner-src/pull/1121/commits/2f285a5cb613efc5e341392d0769efc9562ce330). Feel free to change it as you want.